src/processes.c

```diff
@@ -2112,6 +2120,7 @@ static int ps_read (void)
 	int blocked  = 0;
 	int idle     = 0;
 	int wait     = 0;
+	int num_cmdline = 0;
```

Comment:
Maybe a comment to describe this counter? At first sight I thought we are counting the number of command lines. :P

Reply:
Done. PTAL